src/figuresClasses.ts

```diff
@@ -1,19 +1,69 @@
 export interface Figure {
-
+  shape: string;
+  color: string;
```

Comment:
These types are not strings, create separate types or enums and use them.

a shape (triangle, circle or rectangle);
a color (red, green or blue);
![image](https://github.com/mate-academy/ts_figures_classes/assets/63243136/3b1adf37-dcba-4a68-b9c1-bc401c407298)